docs/usage/interpreter/interpreter_binding_mode.md

```diff
@@ -25,4 +25,97 @@ limitations under the License.
 
 ## Overview
 
+<center><img src="{{BASE_PATH}}/assets/themes/zeppelin/img/docs-img/interpreter_binding_per_note_user.png" height="70%" width="70%"></center>
+<center><img src="{{BASE_PATH}}/assets/themes/zeppelin/img/docs-img/interpreter_binding_scoped_isolated.png" height="70%" width="70%"></center>
 
+<br/>
+
+Interpreter is a JVM process that communicates with Zeppelin daemon using thrift. 
+Each Interpreter process can have a interpreter group, and each interpreter instance belongs to this interpreter group.
```

Comment:
Each interpreter process has a single interpreter group, and this interpreter group can have one or more instances of an interpreter.